shot-android/src/main/java/com/karumi/shot/compose/ComposeScreenshotRunner.kt

```diff
@@ -0,0 +1,21 @@
+package com.karumi.shot.compose
+
+import android.app.Instrumentation
+
+class ComposeScreenshotRunner {
+    companion object {
+
+        lateinit var composeScreenshot: ComposeScreenshot
+
+        fun onCreate(instrumentation: Instrumentation) {
+            composeScreenshot = ComposeScreenshot(
+                    session = ScreenshotTestSession(),
+                    saver = ScreenshotSaver(instrumentation.context.packageName, SemanticsNodeBitmapGenerator())
+            )
+        }
+
+        fun onDestroy() {
```

Comment:
here can you replace {} for = ? or saveMetada return something ?

Reply:
Fixed, I've changed onDestroy to be one-liner.